### ISSUE TITLE
[Enhancement] Redo statisctics modules 

### ISSUE THREAD

**Issue by akasolace** [akasolace/HO#780]
- [ ] Add Club statistics (incl. ranking and power rating)
   - [ ] Clean up existing tab (labels, doublons ... )
   - [ ] Adjust graph colors for all themes

**Comment by akasolace** [comment]
@weisJ in part of the application we are doing graphs and we are using a 10 colors palettes. This unique palette was designed for Nimbus theme and does not work for dark theme. I thought it might make sense to have something like this directly in DarkLaf (e.g. a new category in https://weisj.github.io/darklaf-docs/darcula/defaults_windows.html). What do you think?

**Comment by akasolace** [comment]
@weisJ also I noticed that values in https://weisj.github.io/darklaf-docs/darcula/defaults_windows.html are incorrect. For example:  

%background #21252B is in reality #0E3C4A https://github.com/weisJ/darklaf/blob/deae2142da257b048549f23a5edd3fceb578e50d/theme/src/main/resources/com/github/weisj/darklaf/theme/solarized_dark/solarized_dark_defaults.properties#L31

**Comment by akasolace** [comment]
![image](https://user-images.githubusercontent.com/1136496/99102724-c8659500-25de-11eb-9247-4b93416ea658.png)

**Comment by weisJ** [comment]
> @weisJ in part of the application we are doing graphs and we are using a 10 colors palettes. This unique palette was designed for Nimbus theme and does not work for dark theme. I thought it might make sense to have something like this directly in DarkLaf (e.g. a new category in https://weisj.github.io/darklaf-docs/darcula/defaults_windows.html). What do you think?
> 
>  

Sounds like a reasonable idea. Do you habe a list of the colours which are currently used? Are those colours fixed i.e do they need to have the hue they currently have? I am asking because if I add colour palettes to darklaf I would want to cover as many different colours (hues) with as few colours as possible (adding new colours always takes a bit of time and once they are added it's not that easy to remove or change them dramatically).



> @weisJ also I noticed that values in https://weisj.github.io/darklaf-docs/darcula/defaults_windows.html are incorrect. For example:  
> 
> 
> 
> %background #21252B is in reality #0E3C4A https://github.com/weisJ/darklaf/blob/deae2142da257b048549f23a5edd3fceb578e50d/theme/src/main/resources/com/github/weisj/darklaf/theme/solarized_dark/solarized_dark_defaults.properties#L31
> 

Thank you for noticing. I'll correct them.

**Comment by akasolace** [comment]
@weisJ I started to create 5, 10 and 15 palettes for the various themes.
work is ongoing https://github.com/akasolace/HO/tree/bug/454

**Comment by akasolace** [comment]
Matches Panel

![image](https://user-images.githubusercontent.com/1136496/99109294-e33d0700-25e8-11eb-86be-d914012078ae.png)

**Comment by akasolace** [comment]
linked to #788

**Comment by weisJ** [comment]
@akasolace I'm currently working on a colour palette for darklaf. However I'm not sure I can reliably find a palette for 15 different colours as 10 are about the most I can comfortably create find fitting colours for all themes without colours starting to look too similar. Do you think you could work with that? e.g. by interpolating colours to fill up the palette?

**Comment by akasolace** [comment]
@weisJ I can confirm this is challenging, I gave it a try and I made 3 palettes for respectivly 5, 10 and 15 colors. Indeed some of the colors on the 15 palettes are close one from the other. 
I am not super satisied with what I did so I will clearly utilize Darlak palette for 5 and 10 colors and I will work on improving the 15 colors palettes.
Thank you

**Comment by weisJ** [comment]
I just pushed the first iteration of the colour palette (which should already be quite refined, so no drastic changes are to be expected). I managed to provide 13 colours:
````
palette.yellow
palette.orange
palette.red
palette.pink
palette.purple
palette.indigo
palette.blue
palette.teal
palette.cyan
palette.green
palette.lime
palette.forest
palette.brown
palette.gray
````
The names are only suggestive (and based on the provided default values). This means specifically in the same "group" of colours in different themes the names won't match as good as in others. However they all should be fairly distinct and match the style of the respective theme.
